src/metabase/query_processor/streaming/common.clj

```diff
@@ -112,3 +113,64 @@
       (if (and is-currency? (::mb.viz/currency-in-header merged-settings true))
         (str column-title " (" (currency-identifier merged-settings) ")")
         column-title))))
+
+(defn normalize-keys
```

Comment:
We have `metabase.util/normalize-map` and a few various key normalization functions in the util namespace. Should we consider using a util fn instead? 

I'm happy with this solution too, it's small and straightforward. Just wanted to mention I saw that util fn in case you weren't aware.

Reply:
I'm actually thinking maybe we should drop this entirely and force `metabase.formatter.datetime` to use fully-qualified keys.

Reply:
That's not a bad idea, actually.